runtime/tests/interpreter/memory_metering_test.go

```diff
@@ -118,10 +118,10 @@ func TestInterpretArrayMetering(t *testing.T) {
 		_, err := inter.Invoke("main")
 		require.NoError(t, err)
 
-		assert.Equal(t, uint64(30), meter.getMemory(common.MemoryKindArrayValueBase))
-		assert.Equal(t, uint64(24), meter.getMemory(common.MemoryKindAtreeArrayDataSlab))
-		assert.Equal(t, uint64(3), meter.getMemory(common.MemoryKindAtreeArrayMetaDataSlab))
-		assert.Equal(t, uint64(9), meter.getMemory(common.MemoryKindAtreeArrayElementOverhead))
+		assert.Equal(t, uint64(26), meter.getMemory(common.MemoryKindArrayValueBase))
```

Comment:
Why did the metering change here? It looks like we have fewer arrays/data slabs now

Reply:
Because the outer array is not moved/transferred at the loop expression (i.e: in `for value in values`)